frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -227,47 +232,18 @@ export const VMDisks: React.FC<VMDisksProps> = ({ vmLikeEntity, vmTemplate, vmi
   );
 };
 
-export const VMDisksFirehose: React.FC<VMLikeEntityTabProps> = ({ obj: vmLikeEntity }) => {
-  const vmTemplate = getVMTemplateNamespacedName(vmLikeEntity);
-
-  const resources = [
-    getResource(TemplateModel, {
-      name: vmTemplate?.name,
-      namespace: vmTemplate?.namespace,
-      isList: false,
-      prop: 'vmTemplate',
-    }),
-  ];
-
-  return (
-    <Firehose resources={resources}>
-      <VMDisks vmLikeEntity={vmLikeEntity} />
-    </Firehose>
-  );
-};
-
-export const VMDisksAndFileSystemsPage: React.FC<VMTabProps> = ({
-  obj: vmLikeEntity,
-  vm: vmProp,
-  vmis: vmisProp,
-  vmImports,
-  pods,
-  migrations,
-  pvcs,
-  dataVolumes,
-  customData: { kindObj },
-}) => {
-  const vmTemplate = getVMTemplateNamespacedName(vmLikeEntity);
-
-  const resources = [
-    getResource(TemplateModel, {
-      name: vmTemplate?.name,
-      namespace: vmTemplate?.namespace,
-      isList: false,
-      prop: 'vmTemplate',
-    }),
-  ];
-
+export const VMDisksAndFileSystemsPage: React.FC<VMTabProps> = (props) => {
```

Comment:
please consider making this nested component in his own file

Reply:
https://github.com/openshift/console/pull/7879#discussion_r595350356